.github/workflows/sync-docs-from-node.yml

```
@@ -26,878 +26,342 @@ on:
+      sync_reports: ${{ steps.collect.outputs.all_reports }}
+    steps:
+      - name: Download all sync reports
+        uses: actions/download-artifact@v4
+        with:
+          pattern: sync-reports-*
+          merge-multiple: true
+          path: sync-reports/
+
+      - name: Calculate totals and collect reports
+        id: calculate
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC1009:info:22:7: The mentioned syntax error was in this variable assignment

(shellcheck)